integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java

```diff
@@ -215,6 +227,84 @@ public void testCreateStopStartEnvironment(TestContext testContext) {
         LOGGER.info("Environment stop-start test execution has been finished....");
     }
 
+    @Test(dataProvider = TEST_CONTEXT, timeOut = 9000000)
+    @Description(
+            given = "there is a running cloudbreak",
+            when = "create an attached SDX and Datahubs (in case of AWS, create one of the Datahub with external database)",
+            then = "should be stopped first and started after it, fails to upgrade the datahub cluster " +
+                    "and required services should be in running state in CM")
+    public void testVerticallyScaleDatahubClusterFail(TestContext testContext) {
```

Comment:
- `fails to upgrade` -> `fails to vertically scale`?
- Why do we expect vertical scaling to fail here?